fee_lab_to_nwb/ophys/ophysdatasetnwbconverter.py

```diff
@@ -6,7 +6,7 @@
     PhySortingInterface,
 )
 
-from fee_lab_to_nwb.ophysbehaviorinterface import OphysBehaviorInterface
+from fee_lab_to_nwb.ophys import OphysBehaviorInterface
 
 
 class OphysDatasetNWBConverter(NWBConverter):
```

Comment:
Given there might be other ophys datasets from this lab in the future, maybe we should name this one

```
ScherrerOphysNWBConverter
```

For the extra specification?

Reply:
Yeah I agree, thanks!